### DISCUSSION TITLE
Toggle to use ability on X number of enemies AND/OR boss

### DISCUSSION THREAD

**Discussion by Wicked0neX** [Hekili/hekili#1638] [Ideas]
Right now you can only toggle one or the other. At least as far as I know anyway.

For example: only use convoke if you're fighting 3+ enemies OR you're fighting a boss.

**Comment by marekCs** [comment]
I think it’s up to you. You can see what’s going on (and what’s coming soon) in the game, not the addon. I was playing try hard Raid Mythic runs only with fixed dual mode. When it was a boss, I just did single target, when mobs or 1+ was coming just did Aoe. The perfection of this solution is that the Aoe counts enemies and suggests also appropriate spells for it. I mean it’s different for 3 or 10 mobs in the Aoe window.

**Comment by Hekili** [comment]
This requires a whole change in the "rule structure" for the ability, or that you just make a custom priority in the meantime.  You can submit a feature request ticket and I'll investigate it when there's time, but it's not a priority right now.
